### PR TITLE
Feature: Translate shell folders names & avoid possible bugs

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes N/A

**Change List**
- Now Shell Folders have a tooltip in the sidebar (Before it was empty)
- Path box will show `This PC` instead of the whole Shell folder path
- Prevent paths such as `RecycleBin/..` which will probably cause issues
- Localize Shell Folders names in `JumpList`, `TabName`, `Path box`
- Removed `LocalSettings` translation of `RecycleBin`

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [x] Did you implement any design changes to an existing feature?
   - [x] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open app
   2. Navigate to shell folders

### CODE REVIEW

File: src/Files.App/Filesystem/StorageFileHelpers/StorageFileExtensions.cs [hunk 1]
```diff
@@ -389,6 +395,20 @@ private static string ResolvePath(string path, bool isFtp)
 			return pathBuilder.ToString();
 		}
 
+		private static string ResolveShellPath(string shPath)
```
Comment (yaira2) [inline src/Files.App/Filesystem/StorageFileHelpers/StorageFileExtensions.cs:398]: Should this be a separate helper?

Comment (yaira2) [reply src/Files.App/Filesystem/StorageFileHelpers/StorageFileExtensions.cs:398]: Making a separate helper will allow us to reuse the code.

File: src/Files.App/ViewModels/Settings/PreferencesViewModel.cs [hunk 2]
```diff
@@ -605,9 +605,16 @@ public string Text
 				if (Path == "Home")
 					return "Home".GetLocalizedResource();
 
-				return (Path == CommonPaths.RecycleBinPath)
-					   ? ApplicationData.Current.LocalSettings.Values.Get("RecycleBin_Title", "Recycle Bin")
-					   : Path;
+				if (Path.Equals(CommonPaths.RecycleBinPath, StringComparison.OrdinalIgnoreCase))
```
Comment (yaira2) [inline src/Files.App/ViewModels/Settings/PreferencesViewModel.cs:608]: Can you use the helper?

Comment (ferrariofilippo) [reply src/Files.App/ViewModels/Settings/PreferencesViewModel.cs:608]: This is actually different from the helper I've just pushed. I'll add another one

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**